### DISCUSSION TITLE
Discord Presence "Hide world details in private" does nothing

### DISCUSSION THREAD

**Discussion by SouljaVR** [vrcx-team/VRCX#377] [General]
Not sure if this is just me, but the "Hide world details in public" toggle for the VRCX Discord RPC does nothing for me. No matter what the toggle is set to, it will just say "Private". I would like it to show Instance type/player count no matter what my in game status is, assuming Hide world details is set to OFF. Thank you :)

**Comment by Natsumi-sama** [comment]
Ask Me and Do Not Disturb statuses overwrite this setting.
I'll change details to only be hidden with DnD status.

**Reply by SouljaVR** [reply]
Thank you. If possible a toggle for 'always shown' could be added, so it shows this status no matter what. That is your choice of course but the flexibility would be nice! thank you :)

**Reply by Natsumi-sama** [reply]
I've changed it so "Hide world details" will hide details with ask me status but DnD will be always hidden, I think this makes the most sense without adding extra toggles.

**Reply by SouljaVR** [reply]
> I've changed it so "Hide world details" will hide details with ask me status but DnD will be always hidden, I think this makes the most sense without adding extra toggles.

This works for me, thank you very much for doing this so fast! :)
